src/AaveV3StrategicAssets_20230606/StrategicAssetsManager.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {Initializable} from 'solidity-utils/contracts/transparent-proxy/Initializable.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+
+import {LSDLiquidityGaugeManager} from './LSDLiquidityGaugeManager.sol';
+import {VeTokenManager} from './VeTokenManager.sol';
+import {VlTokenManager} from './VlTokenManager.sol';
+
+contract StrategicAssetsManager is
```

Comment:
Missing natspec. I would also add the author name on all the contracts

Reply:
done